medusa/process_tv.py

```diff
@@ -34,650 +35,640 @@
 shutil.copyfile = shutil_custom.copyfile_custom
 
 
-class ProcessResult(object):  # pylint: disable=too-few-public-methods
+class ProcessResult(object):
+
     def __init__(self):
         self.result = True
-        self.output = ''
+        self.output = u''
         self.missedfiles = []
-        self.aggresult = True
+        self.succeeded = True
 
+        self.video_files = []
```

Comment:
What was your plan with making these properties? Does it make sense to do it as of now?